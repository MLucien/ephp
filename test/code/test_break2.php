<?php

$a = [1,2,3,4,5,6,7,8,9,10];
$b = ["one", "two"];

foreach ($b as $k) {
    foreach ($a as $i)
        foreach ($a as $j) {
            if ($i * $j > 8) {
                break 2;
            }
            print "($k) $i x $j = " . ($i * $j) . "\n";
        }
}

print "OK!\n";
