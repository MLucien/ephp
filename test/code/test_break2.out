(one) 1 x 1 = 1
(one) 1 x 2 = 2
(one) 1 x 3 = 3
(one) 1 x 4 = 4
(one) 1 x 5 = 5
(one) 1 x 6 = 6
(one) 1 x 7 = 7
(one) 1 x 8 = 8
(two) 1 x 1 = 1
(two) 1 x 2 = 2
(two) 1 x 3 = 3
(two) 1 x 4 = 4
(two) 1 x 5 = 5
(two) 1 x 6 = 6
(two) 1 x 7 = 7
(two) 1 x 8 = 8
OK!
